src/types.ts

```diff
@@ -252,10 +252,9 @@ export interface UIDisputeTokens {
 }
 
 export interface StakeDetails extends NormalizedPayout {
-  totalStake: string;
-  completedStake: string;
+  stakeCompleted: string;
   size?: string;
```

Comment:
shouldn't there always be a size? here is nullable.

Reply:
No size if it is the current tentativewinning